app/controllers/sessions_controller.rb

```diff
@@ -4,9 +4,15 @@ def new; end
   def create
     user = User.find_by email: params[:session][:email].downcase
     if user&.authenticate(params[:session][:password])
-      log_in user
-      params[:session][:remember_me] == "1" ? remember(user) : forget(user)
-      redirect_back_or user
+      if user.activated?
+        log_in user
+        params[:session][:remember_me] == "1" ?
+          remember(user) : forget(user)
```

Comment:
toán tử ngôi số 3 chỉ viết trên cùng 1 line chú

Reply:
Đã fix ạ